test/unit/test_type.c

```diff
@@ -455,6 +455,63 @@ static bool test_const_types(void) {
 	mu_end;
 }
 
+static char *array = "int a[65][5][0]";
+static char *array_exp1 = "int [65][5][0]";
+static char *array_ptr = "int * const *a[][][][9]";
+static char *array_ptr_exp1 = "int * const *[0][0][0][9]";
+static char *array_ptr_exp2 = "int * const *a[0][0][0][9]";
```

Comment:
Maybe stupid question from me, but is `a[][][][9]` really equal to `a[0][0][0][9]`?

Reply:
It's VLA, we need some value in the array size anyway, we set 0 when not specified in `[]` during the parsing. Then yes, they are equivalent.